packages/components/date-picker/src/date-picker-com/basic-date-table.vue

```diff
@@ -244,9 +265,24 @@ export default defineComponent({
       return rows_
     })
 
+    watch(
+      () => props.date,
+      async () => {
+        if (tbodyRef.value?.contains(document.activeElement)) {
+          await nextTick()
+          currentCellRef.value?.focus()
+        }
+      },
+      { immediate: true }
+    )
```

Comment:
**Suggestion:** let's move this section into `onMounted` hook since `tbodyRef` will only be instantiated when `onMounted` is invoked. We can avoid unnecessary initial evaluation.

Reply:
Note that this is not a **Blocking** comment, you may choose to leave it as-is.

Reply:
I removed immediate which has the same effect, thanks for the catch.

Reply:
I removed immediate which has the same effect, thanks for the catch.